build.sbt

```diff
@@ -80,10 +80,12 @@ lazy val cobolConverters = (project in file("cobol-converters"))
     // No need to publish this artifact since it has test only at the moment
     publishArtifact := false,
     publish := {},
-    publishLocal := {},
+    publishLocal := {}
+  ).dependsOn(cobolParser)
```

Comment:
The parser contains some generated code. Is it possible to exclude
```
cobol-parser/src/main/scala/za/co/absa/cobrix/cobol/parser/antlr/*.java
```
from coverage checks (for both overall and changed files) ?

Reply:
*.java files should be excluded already. Current target of code-coverage are *.scala classes.
But yes, there is possible to define excludes. See code in build.sbt for prepared example:
`lazy val commonJacocoExcludes: Seq[String] = Seq(
//  "za.co.absa.cobrix.spark.cobol.reader.FixedLenTextReader*", // class and related objects
//  "za.co.absa.cobrix.spark.cobol.reader.RowHandler" // class only
)
`